doc/whats_new.rst

```diff
@@ -25,6 +25,8 @@ Changelog
 
 - Add :func:`mne.io.read_raw_fieldtrip`, :func:`mne.read_epochs_fieldtrip` and :func:`mne.read_evoked_fieldtrip` to import FieldTrip data. By `Thomas Hartmann`_ and `Dirk Gütlin`_.
 
+- Add ``rank`` parameter to :func:`mne.compute_covariance`, :func:`mne.cov.regularize` and related functions to preserve data rank during regularization by `Eric Larson`_ and `Denis Engemann`_
+
```

Comment:
Can you also add that we now support faster low-rank (PCA-space) computation?

Reply:
Pushed a commit, if it's not what you want feel free to give me a more complete suggestion or push a change

I avoided the word "PCA" because it's more about dealing with rank deficiency rather than projecting onto (orthogonal) directions of highest variance

Reply:
One could still highlight that the option can make things significantly faster for SSSd data.

Reply:
"and speed up computation" is already in there